test/mpi/test_mpi.h

```diff
@@ -23,10 +23,22 @@ END_C_DECLS
 #endif
 
 #define STR(x) #x
-#define UCC_CHECK(_call)                                            \
-    if (UCC_OK != (_call)) {                                        \
-        std::cerr << "*** UCC TEST FAIL: " << STR(_call) << "\n";   \
-        MPI_Abort(MPI_COMM_WORLD, -1);                              \
+#define UCC_CHECK(_call)                                                \
+    if (UCC_OK != (_call)) {                                            \
+        std::cerr << "*** UCC TEST FAIL: " << STR(_call) << "\n";       \
+        MPI_Abort(MPI_COMM_WORLD, -1);                                  \
+    }
+
+#define UCC_CHECK_SKIP(_call, _skip_cause)                              \
+    {                                                                   \
+        ucc_status_t status;                                            \
+        status = (_call);                                               \
+        if(UCC_ERR_NOT_SUPPORTED == status)  {                          \
+            _skip_cause = TEST_SKIP_NOT_IMPL_INPLACE;                   \
```

Comment:
lets call it TEST_SKIP_NOT_SUPPORTED (there can be another reasons when UCC returns not supported, e.g. - not supported datatype)

Reply:
renamed